actors/util/smoothing/alpha_beta_filter_test.go

```diff
@@ -65,12 +65,16 @@ func TestCumSumRatioProjection(t *testing.T) {
 	// Note 2: since both methods are approximations with error this is not a
 	// measurement of analytic method's error, it is a sanity check that the
 	// two methods give similar results
-	errBound := big.NewInt(300)
+	errBound := big.NewInt(350)
```

Comment:
@ZenGround0 I needed to change this. Does this means the new constants are off?

Reply:
no its to be expected.  thanks for checking!